[2023-07-25 22:21:54,159] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_exchange_rates_dag.fetch_exchange_rates_task manual__2023-07-25T22:21:53.273961+00:00 [queued]>
[2023-07-25 22:21:54,166] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_exchange_rates_dag.fetch_exchange_rates_task manual__2023-07-25T22:21:53.273961+00:00 [queued]>
[2023-07-25 22:21:54,167] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-25 22:21:54,168] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-25 22:21:54,168] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-25 22:21:54,181] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): fetch_exchange_rates_task> on 2023-07-25 22:21:53.273961+00:00
[2023-07-25 22:21:54,185] {standard_task_runner.py:52} INFO - Started process 14599 to run task
[2023-07-25 22:21:54,188] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fetch_exchange_rates_dag', 'fetch_exchange_rates_task', 'manual__2023-07-25T22:21:53.273961+00:00', '--job-id', '297', '--raw', '--subdir', 'DAGS_FOLDER/my_calc_rates.py', '--cfg-path', '/tmp/tmp7mhqt6jx', '--error-file', '/tmp/tmpmkr6imol']
[2023-07-25 22:21:54,190] {standard_task_runner.py:80} INFO - Job 297: Subtask fetch_exchange_rates_task
[2023-07-25 22:21:54,254] {task_command.py:371} INFO - Running <TaskInstance: fetch_exchange_rates_dag.fetch_exchange_rates_task manual__2023-07-25T22:21:53.273961+00:00 [running]> on host 661c8d62a693
[2023-07-25 22:21:54,340] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=marina_z
AIRFLOW_CTX_DAG_ID=fetch_exchange_rates_dag
AIRFLOW_CTX_TASK_ID=fetch_exchange_rates_task
AIRFLOW_CTX_EXECUTION_DATE=2023-07-25T22:21:53.273961+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-25T22:21:53.273961+00:00
[2023-07-25 22:21:54,683] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/my_calc_rates.py", line 85, in fetch_exchange_rates
    cursor.execute(create_table_query)
NameError: name 'cursor' is not defined
[2023-07-25 22:21:54,693] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=fetch_exchange_rates_dag, task_id=fetch_exchange_rates_task, execution_date=20230725T222153, start_date=20230725T222154, end_date=20230725T222154
[2023-07-25 22:21:54,703] {standard_task_runner.py:97} ERROR - Failed to execute job 297 for task fetch_exchange_rates_task (name 'cursor' is not defined; 14599)
[2023-07-25 22:21:54,723] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-07-25 22:21:54,748] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
